frontend/src/components/Sidebar/DistrictMap.tsx

```diff
@@ -275,12 +276,28 @@ export default function DistrictMap(): JSX.Element {
               legendRef.current = legend;
             }}
             min={0}
-            max={legend.isNormalized ? aggregatedMax : legend.steps[legend.steps.length - 1].value}
+            // used math.round to convert the number in integers and used +1 to avoid having legend max value as 0
+            max={
+              legend.isNormalized
+                ? Math.round(aggregatedMax) + 1
+                : Math.round(legend.steps[legend.steps.length - 1].value) + 1
+            }
```

Comment:
This is also part of a different PR right?
```suggestion
            max={legend.isNormalized ? aggregatedMax : legend.steps[legend.steps.length - 1].value}
```